Tools/TestWebKitAPI/Tests/JavaScriptCore/glib/TestJSC.cpp

```diff
@@ -3905,6 +3905,87 @@ static void testJSCExceptions()
     }
 }
 
+struct foo {
```

Comment:
```suggestion
struct Foo {
```

Reply:
But maybe we can use a better name than Foo... Something like AsyncPromiseTestData?